package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "barnard59-rdf",
-  "version": "0.1.1",
+  "version": "0.2.0",
```

Comment:
Bumping `minor` denotes a breaking change. I don't think this a breaking change. It only be breaking in `-base`

Reply:
Don't know how I managed to have an older master branch, when I released version 0.2.0. Anyway rebased this branch, so no more version change in this PR and the new version will be 0.2.1.